package.json

```diff
@@ -8,7 +8,7 @@
     "start": "nuxt start",
     "generate": "nuxt generate",
     "lint": "npx eslint --ext .vue  --ext .js .",
-    "prepare": "husky install",
+    "lint-fix": "npx eslint --ext .vue  --ext .js . --fix",
```

Comment:
was this done by accident? the `lint-fix` script looks cool, but that should done in a separate PR. And we probably don't want to remove the `prepare` script unless we know it's not needed.